L1Trigger/TrackFindingTracklet/python/Demonstrator_cfi.py

```diff
@@ -0,0 +1,11 @@
+# configuration of Demonstrator.
+import FWCore.ParameterSet.Config as cms
+
+TrackTriggerDemonstrator_params = cms.PSet (
+
+  LabelIn  = cms.string( "TrackFindingTrackletProducerTBout"  ), #
+  LabelOut = cms.string( "TrackFindingTrackletProducerKFin" ), #
+  DirIPBB  = cms.string( "/heplnw039/tschuh/work/proj/kfin/" ), # path to ipbb proj area
```

Comment:
can you clarify what this is doing? it seems more of a test config than a central "python" config?

Reply:
this config tells AnalyzerDemonstrator the start and endpoint of your demonstration as well as the location of the corresponding ipbb project (modeslsim project of f/w corresponding to start- and endpoint).

Reply:
All python cfg files, such as Demonstrator_cfi.py must have a comment at the top explaining what they are. e.g. That this is used to compare FW with SW for the subset fo the chain between LabelIn & LabelOut. And that the FW must be wrapped by EMP & compiled with IPBB.